engine/src/test/java/org/camunda/bpm/engine/test/api/history/removaltime/batch/helper/BatchSetRemovalTimeRule.java

```diff
@@ -172,7 +172,9 @@ public class TestProcessBuilder {
     protected static final String PROCESS_KEY = "process";
     protected static final String ROOT_PROCESS_KEY = "rootProcess";
 
-    ProcessBuilder builder = Bpmn.createExecutableProcess(PROCESS_KEY);
+    ProcessBuilder builder = Bpmn.createExecutableProcess(PROCESS_KEY)
+        .camundaHistoryTimeToLiveString(null);
```

Comment:
❓Does it work with `.camundaHistoryTimeToLiveString(null)`? IIRC, during our meeting you mentioned some issue with this.

Reply:
Debugging through the test, the value that is used effectively for `camundaHistoryTimeToLiveString` will be `null`.

There is only one case where the initialisation to null does not work if not done at  `createExecutableProcess ` and that is during `HistoryCleanupRemovalTimeTest#shouldCleanupProcessInstanceWithoutTTLWithConfigDefault`.